### ISSUE TITLE
ChoiceGroup: accessibilty bug for radio button toggle pattern

### ISSUE THREAD

**Issue by veloiya** [OfficeDev/office-ui-fabric-react#6801]
### Bug Report

- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Products/sites affected: Azure devops

#### Describe the issue:

Application: MicrosoftEdgeCP.exe has the following accessibility issue that needs investigation.

![18c488d7-b0e1-4060-8169-0868c6ce4934](https://user-images.githubusercontent.com/30428533/47356635-472ec300-d6e2-11e8-87a7-5017582f2d01.png)

Issue Details
An element of the given type should not support the Toggle pattern.
An element of the given type should not support the Toggle pattern.

Element path: radio button "Pie chart"

To Reproduce and How to Fix guidance: Please open the attached Keros Test File (*.k4dtest) to reproduce the error in Keros for Desktop.

This accessibility issue was found using Keros for Desktop, a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at https://aka.ms/K4D.

**Comment by cliffkoh** [comment]
This does not repro on Keros for Web but I can sort of repro on Keros for Desktop (actually, all the choice groups in the page have the same warning).

It seems to be complaining about the fact that the element has ToggleState property on it, but that is a property that came inherently from it being a `<input type="radio" />`.  This might require some following up as to what, if anything is actually actionable on Fabric's part.

cc: @micahgodbolt @JasonGore @aneeshack4

**Comment by cliffkoh** [comment]
Synced with @aneeshack4. She will be investigating this.

**Comment by KevinTCoughlin** [comment]
FYI @aneeshack4 @cliffkoh I commented on what I _think_ is a related PR earlier today, please see #6785

**Comment by aneeshack4** [comment]
![image](https://user-images.githubusercontent.com/4161791/47530778-694e4f80-d860-11e8-9579-c65e3e5f2f4c.png)
keros test results file type not supported in github comments but above screenshot shows no failures...

**Comment by veloiya** [comment]
I can still see the error on https://developer.microsoft.com/en-us/fabric#/components/choicegroup.
It shows the error for all the choiceGroups on the page on running test for entire app by Shift+F8.
![image](https://user-images.githubusercontent.com/30428533/47545344-6f4c3b00-d909-11e8-9b45-5243ca5a69ed.png)

**Comment by aneeshack4** [comment]
(deleted keros screenshots bc it's not public yet)

**Comment by veloiya** [comment]
The issue does not repro on Keros for Web anymore.

**Comment by aneeshack4** [comment]
@veloiya Since this no longer repros I'm closing it but if you see any other keros problems in the future, def let us know!
